### DISCUSSION TITLE
20211015 Not possible to select an I2S audio device

### DISCUSSION THREAD

**Discussion by chipdipru** [rern/rAudio#567] [Fixed Issues]
After update 20211015. It is not possible to select an I2S audio device. The switch disappears when you move it.
Raspberry Pi 3 Model B
![1](https://user-images.githubusercontent.com/43340836/137742477-986fcb12-8d45-4393-8c16-60b157f416e9.jpg)

**Comment by praveensg8** [comment]
Iam also facing the smae issue.Iamusing Allo Katana dac with isolator board.Not able to select I2s to on.The switch disappears when you move it.
Raspberry Pi 3 Model B

**Comment by chipdipru** [comment]
done in 20211020

**Reply by praveensg8** [reply]
Thanks but after update to 20211020. It worked but  when taking settings .Iam getting following error:
Errors: Unexpected token ] in JSON at position 1638
{ "page" : "system" , "cpuload" : "3.37 1.74 0.75" , "cputemp" : 62.3 , "startup" : "7s (kernel) + 39s (userspace) " , "throttled" : "0x80008" , "time" : "06:13:44 2021-10-21" , "timezone" : "Asia/Kolkata" , "uptime" : "12m" , "uptimesince" : "2021-10-21 06:01" , "audioaplayname" : "allo-katana-dac-audio" , "audiooutput" : "Allo Katana DAC" , "bluetooth" : true , "bluetoothconf" : [ true, false ] , "firmware" : "20211004-1" , "hostapd" : false, "hostname" : "rAudioKATANA" , "kernel" : "5.10.74-1-raspberrypi4-ARCH aarch64" , "lcd" : false, "lcdchar" : false , "lcdcharaddr" : "0x27 0x3F" , "lcdcharconf" : [ 20,"A00","i2c","0x27","PCF8574",15,18,16,21,22,23,24,false ] , "list" : [ {"icon":"microsd","mountpoint":"/","mounted":true,"source":"/dev/mmcblk0p2","size":"2.9GB/30GB"},{"icon":"usbdrive","mountpoint":"/mnt/MPD/USB/?","mounted":false,"source":"/dev/sda1"},{"icon":"usbdrive","mountpoint":"/mnt/MPD/USB/ADATA HD710 PRO","mounted":true,"source":"/dev/sda2","size":"1.5TB/1.9TB"},{"icon":"networks","mountpoint":"/mnt/MPD/NAS/ALLMUSIC","mounted":false,"source":"//192.168.1.80/ALLMUSIC"} ] , "lcdmodel" : "tft35a" , "mpdoled" : false , "mpdoledconf" : 6 , "online" : true , "ntp" : "pool.ntp.org" , "powerbutton" : false, "powerbuttonconf" : [ 5,40,5 ] , "relays" : false , "rpimodel" : "Raspberry Pi 3 Model B Plus Rev 1.3" , "soc" : "BCM2837B0" , "soccpu" : "4" , "socram" : "952MiB" , "socspeed" : "1400" , "soundprofile" : false , "soundprofileconf" : [ 18000000 ,60 ,1500 ,1000 ] , "version" : "1" , "versionui" : 20211020 , "vuled" : false , "vuledconf" : [ 14,15,18,23,24,25,8 ] , "wlan" : false, "wlanconf" : [ "IN" , ▮] , "wlanconnected" : false}

**Reply by rern** [reply]
Try:
Addons > rAudio 1 > long-press `Update` > ok for `UPDATE` branch
